### PR TITLE
[DISKPART] Fixed Volume Number & Label Issue

### PR DESCRIPTION
**Pull request by sahilbest999**
## Purpose

To Fix Volume Number & Label Issue to protect table layout of diskpart's `list volume` command

JIRA issue: [CORE-18138](https://jira.reactos.org/browse/CORE-18138)

## Proposed changes

- Made Volume label to output only 11 characters at max
- Made Volume numbering left padded

### CODE REVIEW

**Comment by HBelusca** [comment]
Can you show us these improvements in action (with a screenshot)?

File: base/system/diskpart/lang/en-US.rc [hunk 1]
```diff
@@ -51,7 +51,7 @@ BEGIN
     IDS_LIST_PARTITION_NO_DISK "\nThere is no disk to list partitions.\nPlease select a disk and try again.\n\n"
     IDS_LIST_VOLUME_HEAD "  Volume ###  Ltr  Label        FS     Type        Size     Status   Info\n"
     IDS_LIST_VOLUME_LINE "  ----------  ---  -----------  -----  ----------  -------  -------  --------\n"
-    IDS_LIST_VOLUME_FORMAT "  Volume %3lu   %c   %-11s  %-5s  %10u  %4I64u %-2s\n"
+    IDS_LIST_VOLUME_FORMAT "  Volume %-3lu   %c   %-11.11s  %-5s  %10u  %4I64u %-2s\n"
```
Comment (HBelusca) [inline base/system/diskpart/lang/en-US.rc:54]: Regarding the `%-11.11s` : isn't this equivalent to `%-.11s` ?

**Comment by sahilbest999** [comment]
> Can you show us these improvements in action (with a screenshot)?

File: base/system/diskpart/lang/en-US.rc (hunk 1 above)
Comment (binarymaster) [reply base/system/diskpart/lang/en-US.rc:54]: Yes it is:

image

Comment (HBelusca) [reply base/system/diskpart/lang/en-US.rc:54]: So yes, this could be used instead for simplification purposes.

**Comment by sahilbest999** [comment]
> Can you show us these improvements in action (with a screenshot)?

sorry, forgot about that, here's the screenshot

https://i.imgur.com/YqjYyc8.png

**Comment by HBelusca** [comment]
image

File: base/system/diskpart/lang/en-US.rc (hunk 1 above)
Comment (sahilbest999) [reply base/system/diskpart/lang/en-US.rc:54]: image

Comment (HBelusca) [reply base/system/diskpart/lang/en-US.rc:54]: ok then restore the thing 😅

**Review by learn-more — CHANGES_REQUESTED** [review]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email you@example.com
```

Regards,

**Comment by binarymaster** [comment]
> I've amended the commit and changed my name, hopefully this won't happen again

Last thing to fix, we're using names starting with a capital letter :smile: 

image

Should be **Sahil Shahane** - I'd recommend to fix it in your [GitHub profile settings](https://github.com/settings/profile) too.

**Comment by sahilbest999** [comment]
> > I've amended the commit and changed my name, hopefully this won't happen again
> 
> Last thing to fix, we're using names starting with a capital letter smile
> 
> image
> 
> Should be **Sahil Shahane** - I'd recommend to fix it in your [GitHub profile settings](https://github.com/settings/profile) too.

done!

**Review by binarymaster — APPROVED** [review]

**Review by HBelusca — APPROVED** [review]

**Comment by HBelusca** [comment]
Thanks for your PR!

**Pull request merged**